api/resources_portal/models/attachment.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+
+class Attachment(models.Model):
+    class Meta:
+        db_table = "attachments"
+        get_latest_by = "created_at"
+
+    objects = models.Manager()
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    filename = models.TextField(blank=False, null=True, help_text="The name of the attachment.")
+
+    s3_bucket = models.CharField(max_length=255, blank=True, null=True)
+    s3_key = models.CharField(max_length=255, blank=True, null=True)
+
```

Comment:
It might still be helpful to see an attachment description actually. Unless people are meticulous with their filenames as descriptors... It's not in the designs but I'll @dvenprasad for input.

Reply:
not for mvp